modules/EvseManager/Charger.cpp

```diff
@@ -1050,17 +1054,46 @@ void Charger::stopSession() {
     signalEvent(types::evse_manager::SessionEventEnum::SessionFinished);
 }
 
-void Charger::startTransaction() {
+bool Charger::startTransaction() {
     std::lock_guard<std::recursive_mutex> lock(stateMutex);
     stop_transaction_id_tag.clear();
     transaction_active = true;
+
+    // TODO(ddo) client_id, tariff_id, cable_id and user_data are currently not
```

Comment:
client_id we already get from Auth   via this command:

authorize_response:
    arguments:
      provided_token:

cable_id and tarrif_id I think we do not have yet somewhere

Reply:
sorry I don't get it. None of the messages defines currently `client_id` - and the function call does not sound like it helps here 
>   authorize_response:
    description: >-
      Reports the result of an authorization request to the EvseManager. 
      Contains the provided_token for which authorization was requested and
      the validation_result

can you elaborate?

Reply:
Ah you are using the id_token.id_token already which is the same as client_id. So we just need to remove "client_id" from the TODO comment as it is already done.